src/Components/Books/Books.js

```diff
@@ -56,23 +58,39 @@ class Books extends Component {
     })
   }
 
+  searchBooks = (search) => {
+    let findBooks = this.state.books.filter(book => {
+      if (book.title.includes(search) || book.author.includes(search)) {
+        this.setState({foundBooks: [book]})
+      }
+    })
+    return findBooks;
```

Comment:
Is this return necessary? I would think that since the state of foundBooks has been set, we would just reference this.state.foundBooks but I'm not sure.